Sources/Receipt/ReceiptDetailTableViewController.swift

```diff
@@ -0,0 +1,122 @@
+//
+// Copyright 2018 - Present Hyperwallet
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software
+// and associated documentation files (the "Software"), to deal in the Software without restriction,
+// including without limitation the rights to use, copy, modify, merge, publish, distribute,
+// sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all copies or
+// substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+// BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import HyperwalletSDK
+
+public final class ReceiptDetailTableViewController: UITableViewController {
+    private let registeredCells: [(type: AnyClass, id: String)] = [
+        (ReceiptTransactionTableViewCell.self, ReceiptTransactionTableViewCell.reuseIdentifier),
+        (ReceiptFeeTableViewCell.self, ReceiptFeeTableViewCell.reuseIdentifier),
+        (ReceiptDetailTableViewCell.self, ReceiptDetailTableViewCell.reuseIdentifier),
+        (ReceiptNotesTableViewCell.self, ReceiptNotesTableViewCell.reuseIdentifier)
+    ]
+
+    private var presenter: ReceiptDetailViewPresenter!
+
+    public init(with hyperwalletReceipt: HyperwalletReceipt) {
+        super.init(nibName: nil, bundle: nil)
+        presenter = ReceiptDetailViewPresenter(with: hyperwalletReceipt)
+    }
+
+    // swiftlint:disable unavailable_function
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("NSCoding not supported")
+    }
+
+    override public func viewDidLoad() {
+        super.viewDidLoad()
+        title = "title_receipts_details".localized()
+        titleDisplayMode(.never)
+        setViewBackgroundColor()
+        setupReceiptDetailTableView()
+    }
+
+    private func setupReceiptDetailTableView() {
+        tableView = UITableView(frame: view.frame, style: .grouped)
+        tableView.allowsSelection = false
+        tableView.rowHeight = UITableView.automaticDimension
+        tableView.estimatedRowHeight = UITableView.automaticDimension
+        tableView.separatorInset = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: .greatestFiniteMagnitude)
```

Comment:
This one behaves weird on iOS 10. So maybe try ```
if #available(iOS 11.0, *) {
            tableView.separatorInset = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: .greatestFiniteMagnitude)
            tableView.separatorInsetReference = .fromAutomaticInsets
        } else {
            tableView.separatorStyle = UITableViewCell.SeparatorStyle.none
        }
```

Reply:
<img width="334" alt="Screen Shot 2019-06-07 at 8 28 08 AM" src="https://user-images.githubusercontent.com/29735602/59116714-0b5b0d80-8901-11e9-9c45-0a8ce440cc98.png">

Reply:
fixed for ios 10.0